### PR TITLE
feat(FormField): render dom attributes

### PR DESCRIPTION
**Pull request by adampickeral**
add passed in dom attributes to the rendered control goup in form fields

### CODE REVIEW

**Comment by j00k** [comment]
:ship:

**Comment by willcumbie** [comment]
Yikes! Good fix.

File: src/FormField.jsx [hunk 1]
```diff
@@ -4,21 +4,22 @@ import FormFieldValidationBlock from './FormFieldValidationBlock';
 import classnames from 'classnames';
 
 const FormField = (props) => {
+  const { error, success, label, children, help, inlineValidation, ...rest } = props;
```
Comment (hannahstwinter) [inline src/FormField.jsx]: :+1:

File: src/FormField.jsx [hunk 2]
```diff
@@ -4,21 +4,22 @@ import FormFieldValidationBlock from './FormFieldValidationBlock';
...
-      'success': props.success
+      'error': error,
+      'success': success
     }
   );
 
   return (
-    <div className={ classes }>
-      <label className="rs-control-label">{ props.label }</label>
+    <div {...rest} className={ classes }>
```
Comment (MACSkeptic) [inline src/FormField.jsx]: would you want to also append user provided classes to the `className` here?

in other words, should `<FormField className="x" ...` include `x` + the default class names?

Comment (adampickeral) [inline src/FormField.jsx]: yes, yes you would. i paid no attention to the code above that was using `classnames` and assumed it was already doing that

Comment (adampickeral) [inline src/FormField.jsx]: addressed in #169

**Pull request merged**